addons/mail/tests/test_mail_render.py

```diff
@@ -409,6 +409,96 @@ def test_replace_local_links(self):
             rendered = self.env['mail.render.mixin']._replace_local_links(source)
             self.assertEqual(rendered, expected)
 
+    def test_qweb_static_rendering(self):
```

Comment:
:socks: same here, if we change "static" by a better suffix, don't forget to propagate :)